src/app/modules/verified-content/verified-content.service.ts

```diff
@@ -36,6 +36,12 @@ export const getVerifiedContent = ({
       return getSpVerifiedContent(data)
     case AuthType.CP:
       return getCpVerifiedContent(data)
+    case AuthType.SGID:
```

Comment:
iffy on this returning an error. Should we just constrain the type to be back to the previous implementation; i.e. only allow SP/CP auth types? 

Only add in here when SGID is ready to support verified fields?

EDIT: I see that there is a new `SgidUinFin` key being added? Does that not mean SGID has verified content now?

Reply:
It does... but only for e-mail mode. @liangyuanruo has disallowed sgID for providing verified content in storage mode until we have e2e encryption in-place for fields obtained from eg MyInfo and sgID